src/hg.ts

```diff
@@ -572,6 +572,19 @@ export interface Commit extends Revision {
 	bookmarks: string[];
 }
 
+export interface NativeCommit {
```

Comment:
Seems like we don't want to export this, since it's intended to only be used internally.

Reply:
oops, copy paste!